hy/core/shadow.hy

```diff
@@ -138,11 +138,17 @@
 (defn not-in [x y]
   (not-in x y))
 
+(defn get [thing key1 &rest keys]
+  (for* [k (+ (, key1) keys)]
+    (setv thing (get thing k)))
+  thing)
```

Comment:
Looks like Hy is assigning an anon var because of the if. (even though it's not used). That might slow it down more than the equivalent Python would.

Let's remove the if then. It's not really helping, and it slows down the 2-key case noticeably when I profiled it.